examples/pipelines/fl_cross_silo_basic/config.yaml

```diff
@@ -9,22 +9,43 @@ aml:
 # federated learning parameters
 federated_learning:
   orchestrator:
-    compute: "cpu-cluster"
+    compute: "cpu-cluster-orchestrator"
     datastore: "workspaceblobstore"
 
   silos:
-    - compute: cpu-silo-wus2
-      datastore: workspaceblobstore
-      training_data_path: "azureml:mnist-train-01:1"
-      testing_data_path: "azureml:mnist-test-01:1"
-    - compute: cpu-silo-eus2
-      datastore: workspaceblobstore
-      training_data_path: "azureml:mnist-train-02:1"
-      testing_data_path: "azureml:mnist-test-02:1"
-    - compute: cpu-silo-weu
-      datastore: workspaceblobstore
-      training_data_path: "azureml:mnist-train-03:1"
-      testing_data_path: "azureml:mnist-test-03:1"
+    - compute: cpu-cluster-eastus2
+      datastore: silo_datatore_eastus2_private
+      # in this demo, we're using public data from a url instead
+      training_data:
+        type: uri_file
+        mode: 'download'
+        path: https://azureopendatastorage.blob.core.windows.net/mnist/processed/train.csv
```

Comment:
One reason for wrapping these train/test sets in different data assets for each silo was to have a feel in the AML UI that the data is not shared among them and each silo has its own dataset. I also agree, with the new approach - it is pretty straightforward to handle data. We might need to think about this (maybe in the next iteration).

Reply:
Yes I think we need to provide a "feel" in our demo. I filed an issue #60 to figure out what's the best strategy here. I propose to merge this one and add a PR once we define the best approach.